content/04.results.md

```diff
@@ -125,41 +126,37 @@ We observed 8% (16/195) of tumors to be _C11orf95::RELA_ fusion positive ependym
-D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7), other CNS (N < 5 each).
-Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot.
-Only samples with mutations in the listed genes are shown.](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/figures/pngs/primary_only_oncoprint_landscape.png){#fig:Fig2}
+![**Figure 2. Mutational landscape of PBTA tumors.**Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): low-grade glioma astrocytoma (N = 187), ganglioglioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 131): medulloblastomas (N = 97), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 6), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7), other CNS (N < 5 each).Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown.](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/figures/pngs/primary_only_oncoprint_landscape.png){#fig:Fig2}
 
 
-### Landscape of Mutational Processes
+### Mutational co-occurrence and signatures highlight key oncogenic drivers
 
-This section summarizes the mutational landscape of the pediatric brain tumor samples of this dataset.
-Figure {@fig:openpbta-overview}A shows the tumor mutation burden as compared to adult TCGA brain-related tumors.
-Figure {@fig:mutational-landscape-overview}B-C show concordance of these samples with mutational signatures from COSMIC [@url:https://cancer.sanger.ac.uk/cosmic] and Alexandrov et al, 2013 [@doi:10.1038/nature12477] signature sets.
+The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). 
+We observed _TP53_ to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (35.2%, OR = 32, q = 8.46e-17), _ATRX_ (21.2%, OR = 20, q = 4.43e-8), _NF1_ (14.1%, OR = 8.62, q = 5.45e-5), and _EGFR_ (10%, OR = 18.2, q = 1.6e-4). 
```

Comment:
The percentages here and below are not really defined. I think it might makes sense to briefly define or reference what the calculation is, as I do not think it is obvious or intuitive from the text.

Reply:
>Noting this comment still applies. I might remove the statement about TP53 & PPM1D. While I can see why we might want to keep it, the multiple testing here is significant, you are only reporting the q values for all other sets.

I missed this, thanks - I will update.

Reply:
>The percentages here and below are not really defined. I think it might makes sense to briefly define or reference what the calculation is, as I do not think it is obvious or intuitive from the text.

I see that I only had added calculations for the first percentage, but the percentages are the percent of tumors as described. For example, "We observed _TP53_ to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666)" means that tp53 was mutated in 8.4% of tumors in the OpenPBTA, and the following sentences are part of the stats which support the sentence, so I am not sure how further to define it - do you have any suggestions? I think I had written the oncoprint section in a similar way.

Reply:
Right, so the 8.4% makes sense, as it is a simple percentage of all tumors. But the later percentages for coocurrence are percent of tumors having mutations for both genes, among those with at least one mutation. The confusion is that the meaning of the 8.4% is not the same as the 35.2% that appears next in the same sentence or the other percentages here. 

For mutual exclusivity, the percentage is being calculated differently again, so it should be defined there as well. Alternatively, just the OR may be sufficient.

For TP53 & PPM1D, I might say something like "TP53 & PPM1D mutations have been shown to be mutually exclusive in HGATs, and our data recapitulates that trend, though the result is not significant after multiple testing correction."

Reply:
updated all of this in the latest two commits. @runjin326 will work on adding confidence intervals for the odds ratios, so in the meantime, I clarified the percentages and perhaps we can merge - I will add a ticket to possibly come back to this and reword once the CIs are calculated, if that is OK with you